rocAL/rocAL/include/meta_data/meta_data.h

```diff
@@ -160,95 +194,105 @@ struct MetaDataBatch
         return this;
     }
```

Comment:
Need to check if the above concatenate and += operator is required, And remove it